src/edges_analysis/analysis/sky_models.py

```diff
@@ -91,7 +91,9 @@ def get_index(
         lon: [None, np.ndarray] = None,
         sky_model: [None, SkyModel] = None,
     ) -> np.ndarray:
-        return self.index
+        index = np.zeros(len(lat))
```

Comment:
Can you put a comment on top of this about why you do this? It's not obvious, and I might at some point go and revert it unless there's a comment why I shouldn't :-)